### PR TITLE
docs: Add release notes for Metals 1.3.1

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -9,7 +9,7 @@ Global / onChangedBuildSource := ReloadOnSourceChanges
 Global / resolvers += "scala-integration" at
   "https://scala-ci.typesafe.com/artifactory/scala-integration/"
 
-def localSnapshotVersion = "1.3.1-SNAPSHOT"
+def localSnapshotVersion = "1.3.2-SNAPSHOT"
```
Comment (kasiaMarek) [inline build.sbt:12]: mima?

File: website/blog/2024-05-13-thallium.md [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+give Metals a try!
+
+- [Resolve classpath lazily in Bazel](#resolve-classpath-lazily-in-bazel)
+- [Replace `slowTask` with LSP progress](#replace-slowtask-with-lsp-progress)
+- [No workspace support](#no-workspace-support)
+
+## Resolve classpath lazily in Bazel
+
+Metals uses [Bazel BSP](https://github.com/jetBrains/bazel-bsp/) to interact if
+Bazel itself via the Build Server Protocol. Up until this release, we would
```
Comment (kasiaMarek) [inline website/blog/2024-05-13-thallium.md:56]: Something is wrong with this sentence.

File: website/blog/2024-05-13-thallium.md [hunk 3]
```diff
@@ -0,0 +1,326 @@
...
+This means that editors that did not implement the `slow task` extension will
+now properly show the progress of anything running within Metals. For anything
+that did implement it, there might be some visual changes. In Visual Studio Code
+the progress will be contained at the bottom in the status bar, which can be
+clicked on to see all current progress messages. Since, it will also contain all
+the current compilation, we removed `Ongoing compilations` section from the
+Metals view. This is possible thanks to the great work done by
+([kasiaMarek](https://github.com/kasiaMarek))
+
+## No workspace support
```
Comment (kasiaMarek) [inline website/blog/2024-05-13-thallium.md:80]: Did you unrevert the change in VSCode?

File: website/blog/2024-05-13-thallium.md [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+proper progress endpoint available. This is now used in Metals.
+
+This means that editors that did not implement the `slow task` extension will
+now properly show the progress of anything running within Metals. For anything
+that did implement it, there might be some visual changes. In Visual Studio Code
+the progress will be contained at the bottom in the status bar, which can be
+clicked on to see all current progress messages. Since, it will also contain all
+the current compilation, we removed `Ongoing compilations` section from the
+Metals view. This is possible thanks to the great work done by
+([kasiaMarek](https://github.com/kasiaMarek))
```
Comment (kasiaMarek) [inline website/blog/2024-05-13-thallium.md:78]:
```suggestion
([kasiaMarek](https://github.com/kasiaMarek)).
```

File: website/blog/2024-05-13-thallium.md [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+
+## No workspace support
+
+Thanks to [kasiaMarek](https://github.com/kasiaMarek) it's now possible to start
+Metals for a single file without a workspace. Metals will automatically use
+Scala CLI for such single files to provide as much support as possible including
+standard LSP features as well as running and debugging.
```
Comment (kasiaMarek) [inline website/blog/2024-05-13-thallium.md:85]: I think that debugging is broken before https://github.com/scalameta/metals/pull/6347, so maybe just don't mention it.

**Review by kasiaMarek — COMMENTED** [review]
Shouldn't you also update `mtags-auto-release`?

File: website/blog/2024-05-13-thallium.md (hunk 3 above)
Comment (tgodzik) [reply website/blog/2024-05-13-thallium.md:80]: Will do before the release, thanks for reminding me!

File: build.sbt (hunk 1 above)
Comment (tgodzik) [reply build.sbt:12]: What do you mean?

File: website/blog/2024-05-13-thallium.md (hunk 5 above)
Comment (tgodzik) [reply website/blog/2024-05-13-thallium.md:85]: Removed

File: website/blog/2024-05-13-thallium.md (hunk 2 above)
Comment (tgodzik) [reply website/blog/2024-05-13-thallium.md]: it should have been `with` instead of `if`. Changed!

File: build.sbt (hunk 1 above)
Comment (tgodzik) [reply build.sbt:12]: Ach, you mean mimaPreviousArtifacts? I am thinking we shouldn't actually update it if it works.

File: website/blog/2024-05-14-thallium.md [hunk 6]
```diff
@@ -0,0 +1,338 @@
...
+the current compilation, we removed `Ongoing compilations` section from the
+Metals view. This is possible thanks to the great work done by
+[kasiaMarek](https://github.com/kasiaMarek).
+
+## No workspace support
+
+Thanks to [kasiaMarek](https://github.com/kasiaMarek) it's now possible to start
+Metals for a single file without a workspace. Metals will automatically use
+Scala CLI for such single files to provide as much support as possible including
+standard LSP features. such as completions, hover and more.
```
Comment (kasiaMarek) [inline website/blog/2024-05-14-thallium.md:85]:
```suggestion
standard LSP features, such as completions, hover and more.
```

File: website/blog/2024-05-14-thallium.md [hunk 7]
```diff
@@ -0,0 +1,338 @@
...
+
+## Replace `slowTask` with LSP progress
+
+There are several Language Server Protocol extensions that are used in Metals to
+provide features that go outside the LSP standard. This also included
+`Slow task` extension, but since a couple of LSP releases there has been a
+proper progress endpoint available. This is now used in Metals.
+
+This means that editors that did not implement the `slow task` extension will
+now properly show the progress of anything running within Metals. For anything
```
Comment (kasiaMarek) [inline website/blog/2024-05-14-thallium.md:72]:
```suggestion
now properly show the progress of tasks running within Metals. For anything
```

**Review by kasiaMarek — APPROVED** [review]

**Pull request merged**